### PR TITLE
Update translations Lang ID & add missing key(s)

### PR DESCRIPTION
**Pull request by totoprayogo1916**
The main thing is to add Indonesian. But according to PR #127, asked to run a test. And in order to pass the Travis CI, I added the missing keys and deleted (set as comment) the extra keys.

### CODE REVIEW

File: Language/pt-BR/CLI.php
Comment (natanfelles) [inline Language/pt-BR/CLI.php:41]: You are removing indexes that are in the current CodeIgniter4 repository.
The pt-BR language is synchronized with the new version. Please, remove it.

Comment (natanfelles) [reply Language/pt-BR/CLI.php:41]: Or rather, do not remove it.

Comment (totoprayogo1916) [reply Language/pt-BR/CLI.php:41]: I cant find at En lang 

https://github.com/codeigniter4/framework/blob/master/system/Language/en/CLI.php

**Comment by totoprayogo1916** [comment]
@natanfelles : Please help for Fabricator.php & Seed.php

File: Language/pt-BR/CLI.php
Comment (natanfelles) [reply Language/pt-BR/CLI.php:41]: Found here: https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/Language/en/CLI.php

**Comment by natanfelles** [comment]
@totoprayogo1916 Done at https://github.com/codeigniter4/translations/pull/130

File: Language/pt-BR/CLI.php
Comment (totoprayogo1916) [reply Language/pt-BR/CLI.php:41]: You are correct, but the reference for the current translate is the stable version.

Let's wait for comments / reviews from @lonnieezell

Comment (totoprayogo1916) [reply Language/pt-BR/CLI.php:41]: If tested, the results are required to remove the extra key to pass the test. 
Like comment @lonnieezell at #127.

**Comment by totoprayogo1916** [comment]
@natanfelles I think you create suggestion on this PR

File: Language/pt-BR/Validation.php
Comment (totoprayogo1916) [inline Language/pt-BR/Validation.php:73]:
```suggestion
	'not_equals'            => 'O campo {field} não pode ser: {param}.',
	// 'not_in_list'           => 'O campo {field} não deve ser um desses: {param}.',
```

**Comment by totoprayogo1916** [comment]
Finally, the Travis CI test has passed.
The missing key has been added, but has not been translated.
Make some keys as comments on keys that are still in a develop version.
Now ready to merge. ( on lang pt-BR cc: @natanfelles )
Let me know if anything is missing or needs to be removed.

cc: @lonnieezell

**Comment by totoprayogo1916** [comment]
updated

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**
